### PR TITLE
ENT-5555: Modified cftransport cleanup to avoid errors (3.12.x)

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

**Comment by craigcomstock** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5117/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5117/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5117/

**Comment by craigcomstock** [comment]
Need to restart a fresh build. Packages have expired.
@cf-bottom jenkins please.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5128/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5128/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5128/

File: cfe_internal/enterprise/federation/federation.cf [hunk 1]
```diff
@@ -302,9 +302,13 @@ bundle agent clean_when_off
...
     "rm_rf_cftransport_home_dir" usebundle => default:rm_rf("$(home)");
 
+  classes:
+    selinux_enabled.default:_stdlib_path_exists_semanage::
+      "has_cftransport_fcontext" expression => returnszero("$(paths.semanage) fcontext -l | grep $(home)", "useshell");
+
   commands:
     # _stdlib_path_exists_<command> and paths.<command> are defined is masterfiles/lib/paths.cf
-    selinux_enabled.default:_stdlib_path_exists_semanage::
+    selinux_enabled.default:_stdlib_path_exists_semanage.has_cftransport_fcontext::
```
Comment (nickanderson) [inline cfe_internal/enterprise/federation/federation.cf:311]: redundant, but i think I said the same thing previously.

**Pull request merged**